generator/recipe.go

```diff
@@ -25,10 +26,13 @@ var templateFuncs = map[string]interface{}{
 	"indent": indent,
 }
 
+var recipeVersions = [1]string{"v1beta1"}
```

Comment:
So in this approach, a meteor release will contain an array of recipe versions it can support?
So, in future if we are releasing a meteor that is backward compatible with `v1beta1`, then this array will have multiple versions in it, like `{"v1beta1","v1beta2"}` ?

Reply:
Yes.